tests/system_health/test_system_health.py

```diff
@@ -98,6 +98,7 @@ def test_service_checker(duthosts, enum_rand_one_per_hwsku_hostname):
 @pytest.mark.disable_loganalyzer
 def test_service_checker_with_process_exit(duthosts, enum_rand_one_per_hwsku_hostname):
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    skip_release(duthost, ["201811", "201911", "202012", "202106"])
```

Comment:
Please check fixture "check_image_version" in this file. I suppose the skip should be done there.

Reply:
check_image_version is module scoped and will skip all the tests in this file. Is that the intention? Didn't see any issues with the other testcase test_service_checker on 202012 branch

Reply:
@Junchao-Mellanox Are the versions up-to-date?

Reply:
Hi @qiluo-msft , yes, I see the fix was merged to 202012/202106. 
Hi @neethajohn, this case should not be skipped on 202012/202106, if it cannot pass on latest 202012/202106, let's open an issue.

Reply:
Thank you @Junchao-Mellanox for the clarification. I did see Azure/sonic-buildimage#9367 merged into 202012. The latest 202012 image doesn't have this change. The next build should.